app/migrations/seed.py

```diff
@@ -257,32 +259,32 @@ def update_or_create_menu(name, description, parent_id=None):
     print('------------------------------------------------------------')
     print('Done creating USSD Menus')
 
+    
+    
+    print('Creating Business Categories')
+    print('**********************************************************************')
+    business_categories = [
+        {'name': 'Food', 'icon': 'message'},
+        {'name': 'Water', 'icon': 'message'},
+        {'name': 'Energy', 'icon': 'message'},
+        {'name': 'Education', 'icon': 'message'},
+        {'name': 'Health', 'icon': 'message'},
+        {'name': 'General shop', 'icon': 'message'},
+        {'name': 'Environment', 'icon': 'message'},
+        {'name': 'Transport', 'icon': 'message'},
+        {'name': 'Labour', 'icon': 'message'},
+        {'name': 'Other', 'icon': 'message'},
+    ]
+    for business_category in business_categories:
+        try:
+            usage = TransferUsage(name=business_category['name'], icon=business_category['icon'], 
+            priority=1, default=True)
+            db.session.add(usage)
+            db.session.commit()
+                
+        except IntegrityError as e:
```

Comment:
Because there's no check to see if the category already exists, this looks like it'd hit a 'duplicate name' IntegrityError over and over again if you run the seed more than once?

Perhaps it'd make more sense to: 
Check if a name already exists, and if so, update the date to reflect that's the in seed. This would make it easy to update translations, icons etc. 

Also, it might be good to include a priority in the seed data?